raiden/tests/integration/test_send_queued_messages.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+import gevent
+from raiden import waiting
+from raiden.transfer import views
+from raiden.tests.utils.network import CHAIN
+from raiden.tests.utils.transfer import (
+    direct_transfer,
+    assert_synced_channel_state,
+)
+from raiden.network.transport import MatrixTransport
+from raiden.app import App
+
+
+@pytest.mark.parametrize('in_memory_database', [False])
+@pytest.mark.parametrize('deposit', [10])
+@pytest.mark.parametrize('channels_per_node', [CHAIN])
+@pytest.mark.parametrize('number_of_nodes', [2])
+def test_recovery_happy_case(
```

Comment:
Could add a docstring and change the test name, to explaining this test is for restarts with transfers in the queue?